templates/unstable-vite-cloudflare/functions/[[path]].ts

```diff
@@ -0,0 +1,14 @@
+import { type ServerBuild } from "@remix-run/cloudflare";
+import { createPagesFunctionHandler } from "@remix-run/cloudflare-pages";
+
+const build = async (): Promise<ServerBuild> => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  // eslint-disable-next-line import/no-unresolved
+  return import("../build/server");
+};
```

Comment:
@markdalgleish I'm tempted to design a `getBuild` API that returns `Promise<ServerBuild | undefined>` to avoid this ugly boilerplate. Server bundles complicate things slightly, though we could just return `undefined` when server bundles are present.

Reply:
Writing the above snippet made me think that we should just include the `throw Error` part within `getBuild` and return `Promise<ServerBuild>` instead.

Reply:
That said, this is something we can land after Vite is stabilized.